app/code/Magento/AsynchronousOperations/Controller/Adminhtml/Notification/Dismiss.php

```diff
@@ -6,44 +6,36 @@
 namespace Magento\AsynchronousOperations\Controller\Adminhtml\Notification;
 
 use Magento\AsynchronousOperations\Model\BulkNotificationManagement;
-use Magento\Backend\App\Action\Context;
 use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\Controller\Result\Json;
 use Magento\Framework\Controller\ResultFactory;
 
 /**
  * Class Bulk Notification Dismiss Controller
  */
-class Dismiss extends Action
+class Dismiss extends Action implements HttpGetActionInterface
 {
+    public const ADMIN_RESOURCE = 'Magento_AsynchronousOperations::system_magento_logging_bulk_operations';
```

Comment:
This constant used several times, make sense to unify it?

Reply:
This constant is required for \Magento\Backend\App\AbstractAction::_isAllowed, so it should be defined separately for each action.